### PR TITLE
[Modular] New Lockers! (Skyrat Edition) [MDB IGNORE]

### PR DESCRIPTION
**Pull request by ghost**
## About The Pull Request
#25245 Redone because it deleted my everything when I pushed it. So, I redid it. All of it.

PR is essentially https://github.com/Skyrat-SS13/Skyrat-tg/pull/25167 but for our own lockers, plus some other maintenance and a little surprise.

Full Changelist:
- Removes redundant `/station` subtype of the NTC Locker _(this is the source of all map changes)_
- Makes the formatting on the Barber's, NTC & Blueshield's Lockers/Garment Bags in line with all others _(no more caps, no more \proper)_
- Moves the Sec Locker overrides to the master file
- Removes the overrides on armory lockers, now that TG has cool ones for them that aren't redsec _(they're green and have nifty icons)_
- Resprites the NTC, Blueshield, Sec, Warden, HoS, and Barber Lockers
- **NEW!:** Wall Lockers can now be rebuilt, using 2 iron _(same as building a normal closet, only this one makes a wallmount)_
- Wall Lockers are resprited, stored in their own .dmi so that they can have unique overlays
- Several fixes/tweaks to Wall Lockers to prevent bugs
--- 
~~To-Do: Sort out why the wall lockers have bounce on the animated state.~~ Done, animation is now smooth!

## How This Contributes To The Skyrat Roleplay Experience
I love decrusting I love decrusting I love desouling I love desouling!!!!

## Proof of Testing

<details>
<summary>Screenshots/Videos</summary>
 Security Lockers
  
image

Blueshield
image

NTC
image

Barber (God that vendor needs to be redone)
image

Wall Lockers are now not only up-to-date... (these images are a tinyyy bit outdated tho. I made them like 1px shorter and cleaned the inside's shading a bit)
image

And have working overlays...
image

But can be re-built and mounted!
image

</details>

## Changelog

:cl:
qol: Wall Lockers can now be rebuilt! The wall mounts can be crafted the same as a default closet, with 2 iron in-hand.
image: resprites (almost) all Skyrat lockers, including wall-mounted ones! They should be much more in line with TG's sprites now.
fix: fixed several bugs with wall-mounted lockers, and formatting inconsistencies of modular lockers
/:cl:

### CODE REVIEW

File: modular_skyrat/modules/mapping/code/lockers/interdyne_fob/generic.dm [hunk 1]
```diff
@@ -3,10 +3,6 @@
 	req_access = list("syndicate")
 
 // Generic lockers/subtypes
-/obj/structure/closet/secure_closet/personal
-	icon_door = "cabinet"
-	icon_state = "cabinet"
-
```
Comment (ghost) [inline modular_skyrat/modules/mapping/code/lockers/interdyne_fob/generic.dm:9]: You know, there's explicitly an `/obj/structure/closet/secure_closet/personal/cabinet`. There has been for... 9 years? 11?
The fact this override was ever added is kind of gross!

**Comment by vinylspiders** [comment]
Great work! Especially appreciate the lock overlay actually matching up with the handle on the wallmounts

**Review by vinylspiders — APPROVED** [review]
@GoldenAlpharex any objections?

**Review by GoldenAlpharex — APPROVED** [review]
Looks fine to me!

**Pull request merged**